main.go

```diff
@@ -105,6 +129,51 @@ func loadConfig(app *kingpin.Application, element *kingpin.ParseElement, ctx *ki
 	return err
 }
 
+func findRCFile() (fullPath string, found bool, err error) {
+	prevPath := ""
+	dirPath, err := os.Getwd()
+	if err != nil {
+		return "", false, err
+	}
+
+	for dirPath != prevPath {
+		var fullPath string
+		var found bool
+		fullPath, found, err = findRCFileInDir(dirPath)
+		if err != nil || found {
+			return fullPath, found, err
+		}
+		prevPath, dirPath = dirPath, filepath.Dir(dirPath)
+	}
+
+	homeDir, err := getHomeDir()
```

Comment:
I'm not sure about this one. If we do support a global user config it should probably be under [$XDG_CONFIG_HOME](https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html) (at least on linux/osx). 

We could also add this later as a follow up as well, so I'm tempted to say let's only add the default project config for now.